pkgs/top-level/all-packages.nix

```diff
@@ -35044,7 +35044,11 @@ with pkgs;
   };
 
   qutebrowser = callPackage ../applications/networking/browsers/qutebrowser {
-    inherit (qt6Packages) qtbase qtwebengine wrapQtAppsHook qtwayland;
+    inherit (__splicedPackages.qt6Packages) qtbase qtwebengine wrapQtAppsHook qtwayland;
+  };
+
+  qutebrowser-qt5 = callPackage ../applications/networking/browsers/qutebrowser {
```

Comment:
It needs to have -qt5 appended to the `pname` https://github.com/NixOS/nixpkgs/blob/40e5c0270fab19a0201e9a03791112b90fe09ed9/pkgs/applications/networking/browsers/qutebrowser/default.nix#L29

Because nix-env uses `name` , idk what it does when 2 derivations have the same pname and version

Reply:
I honestly don't care, `nix-env` shouldn't be used. This might be fixed by marking the Qt 5 version insecure though.

Reply:
Added it